crates/environ/src/component/vmcomponent_offsets.rs

```diff
@@ -262,6 +267,20 @@ impl<P: PtrSize> VMComponentOffsets<P> {
         self.runtime_memories() + index.as_u32() * u32::from(self.ptr.size())
     }
 
+    /// The offset of the base of the `runtime_tables` field
+    #[inline]
+    pub fn runtime_tables(&self) -> u32 {
+        self.tables
+    }
```

Comment:
Mind updating the comment at the top of this file with the pseudo-struct definition to include the table section?